srv/salt/state/conda_env_with_dask.sls

```diff
@@ -0,0 +1,33 @@
+# File:/srv/salt/conda_env_with_dask.sls
+
+check_conda_env_py39_dask:
+  cmd.run:
+    - name: 'conda env list | grep "py39_dask"'
+    - shell: /bin/bash
+    - env:
+      - PATH: /root/miniconda3/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin
+    - require:
+      - check_conda_install  # miniconda.sls; will ensure conda is set up.
```

Comment:
A comment or refactor is needed here to add context. doing this to much will make it harder for others to follow along. this one instance is fine with a comment.

Reply:
actually is this necessary at all?

Reply:
What context is needed? Is that comment sufficient? Would a full path to the state be better?

Reply:
Wait, is it necessary?
My understanding is that, to some degree, states should be independent - so if a state relies on a prior condition being satisfied, or another state being true/complete, this should be explicit.

Reply:
Being explicit isnt always necessary If you are coming from the assumption that something should already be on the system outside of salt then you can explicitly check, if its something you know your salt is installing in the highstate then you dont need to be explicit.

The other thing to consider is every time you reference another state or state file you raise the likelihood of a subsequent refactor forgetting to remove or adjust the referencing states that have been tethered to it. This eventually can make testing state a bit of a pain depending on your testing method, as simply removing a state temporarily might require lots of commenting out in other states.

Reply:
Fair enough, I'll remove it. 

It *could* be added as an explicit `check_miniconda_is_installed_before_checking_for_env` but at that point, you could have many different states checking for the same things before doing anything, and I wouldn't want to go there.

Reply:
Do I need a comment pointing out that this state presumes conda is installed at that specific path, or is it obvious?